Presentation/PresentationStyle.swift

```diff
@@ -274,3 +289,81 @@ private class PopoverPresentationControllerDelegate: NSObject, UIPopoverPresenta
 }
 
 private var isIpad: Bool { return UIDevice.current.userInterfaceIdiom == .pad }
+
+public extension UIViewController {
+    var customAdaptivePresentationDelegate: CustomAdaptivePresentationDelegate {
```

Comment:
Todo:

- [ ] Add docs what is this and that when presenting with Presentation you should not set your own `presentationController` delegate